content/_data/adopters/Avoris-Travel.adoc

```diff
@@ -1,6 +1,6 @@
 ---
 name: "Avoris Travel"
 url: https://www.avoristravel.com/
-detailsUrl: https://jenkinsistheway.io/user-story/jenkins-is-the-way-to-do-things-quickly-simply-and-in-a-powerful-way/
+detailsUrl: https://jenkinsistheway.io/case-studies/jenkins-case-study-avoris-travel/
```

Comment:
sooooooooooooonnnnnnnnnnn

Reply:
No rush.  We can do this incrementally.  I'm merging this because it is an improvement along the road.